Sources/Sisyphos/Page Elements/OtherElement.swift

```diff
@@ -0,0 +1,50 @@
+public struct OtherElement: PageElement {
```

Comment:
Sorry for not catching in the first round of reviews: I think we should rename this to `Other` to be more in line of how the other element types work. For all of the other element types, the name is a capitalized version of [XCUIElement.ElementType](https://developer.apple.com/documentation/xctest/xcuielement/elementtype) (see [our documentation](https://github.com/SisyphosTests/Sisyphos/blob/main/Sources/Sisyphos/Documentation.docc/WritingTests.md?plain=1#L24)).

Applying this rule here, it would mean that we need to call this page element `Other`.

Reply:
7d06d8836f0e591192f4dc617ee154bbd0c327e3